### PR TITLE
Complete Issue/#8 setup repository

### PR DESCRIPTION
**Pull request by FreakyF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- [How to run](#How-to-run)
+
+## Description
+
+Weather-track is a simple weather application developed in Java that empowers users to explore real-time weather
+conditions and forecasts for cities worldwide. At the core of this application is an intuitive user
+interface that ensures effortless navigation and interaction.
+The interface is meticulously designed to be visually appealing and user-friendly, allowing users to seamlessly explore
+its various features.
+From selecting a specific city or location to accessing up-to-the-minute weather updates, every aspect of the user
```
Comment (Tymisko) [inline README.md:17]: Do we plan to support `location`?

**Review by Tymisko — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (FreakyF) [reply README.md:17]: I am not entirely certain. I would keep it.

Comment (Tymisko) [reply README.md:17]: @FreakyF Create an issue for this functionality.

Comment (FreakyF) [reply README.md]: @Tymisko i thought about this feature, and we won't support it.

**Pull request merged**